Source/WebCore/platform/gtk/SelectionData.h

```diff
@@ -66,13 +66,16 @@ class SelectionData {
     void clearBuffers() { m_buffers.clear(); }
 
     void setCustomData(Ref<SharedBuffer>&& buffer) { m_customData = WTFMove(buffer); }
-    SharedBuffer* customData() const { return m_customData.get(); }
+    RefPtr<SharedBuffer> customData() const { return m_customData; }
```

Comment:
Why not `RefPtr<SharedBuffer>&` or `const RefPtr<SharedBuffer>&`

Reply:
Thanks!